content/en/tracing/guide/configuring-primary-operation.md

```diff
@@ -82,15 +82,123 @@ By default, the resource name would be set to this as it’s the name of the fun
 
 ## OpenTracing
 
-When using Datadog, the Opentracing operation name is a resource and the Opentracing "component" tag is Datadog's span name. To define (in Opentracing terms) a span that has the resource "/user/profile", and the span name "http.request", use this Go example:
+When using Datadog, the OpenTracing operation name is a resource and the Opentracing "component" tag is Datadog's span name. For example, to define (in OpenTracing terms) a span that has the resource "/user/profile", and the span name "http.request":
 
-```text
+{{< programming-lang-wrapper langs="java,python,ruby,go,nodejs,.NET,php" >}}
+{{< programming-lang lang="java" >}}
+
+
+
+```java
+Span span = tracer.buildSpan("http.request").start();
+
+try (Scope scope = tracer.activateSpan(span)) {
+    span.setTag("service.name", "service_name");
+    span.setTag("resource.name", "/user/profile");
+    // code being traced
+} finally {
+    span.finish();
+}
+
+```
+
+For more information on how to set up Java and OpenTracing, refer to this [article][3].
+
+{{< /programming-lang >}}
+{{< programming-lang lang="python" >}}
+
+```python
+  span = opentracing.tracer.start_span(‘http.request')
```

Comment:
```suggestion
  span = opentracing.tracer.start_span('http.request')
```

Reply:
Thanks! I ended up resolving that separately since I had that issue with PHP with the quotes :) .